RxSwift/Schedulers/SerialDispatchQueueScheduler.swift

```diff
@@ -38,6 +38,12 @@ public class SerialDispatchQueueScheduler : SchedulerType {
 
     let configuration: DispatchQueueConfiguration
     
+    /**
+    Constructs new `SerialDispatchQueueScheduler` that wraps `queue`.
```

Comment:
One last nit-pick, it should be "that wraps `serialQueue`". I think I'm good beyond that.

Reply:
@freak4pc What about https://github.com/Cee/RxSwift/blob/9daf8e437136c1486b2ed6391bb91e86b8b0840a/RxSwift/Schedulers/ConcurrentDispatchQueueScheduler.swift#L26

Reply:
The argument is `queue` in that link. Here its `serialQueue` that's why I mentioned it.

Reply:
oh okay then